### PR TITLE
Update Guides “contributing”

### PR DESCRIPTION
**Pull request by james**
Automatically generated by Netlify CMS

### CODE REVIEW

File: _guides/contributing.md [hunk 1]
```diff
@@ -1,143 +1,35 @@
...
-
-### It’s in guides if
-
-* it’s a detailed, step by step instruction.
-* we’re talking about something specialist. This could mean we are using jargon
-  and referring to community-known methods and ways of working.
+This playbook is public and anyone, inside or outside of dxw, can suggest changes to it. This means you!
 
-### It’s something else if
+We manage changes via Github 'Pull Requests'. If you're experiences in doing this, then all you need to know is that this repository is available at https://github.com/dxw/playbook2
```
Comment (rjw1) [inline _guides/contributing.md:6]:
```suggestion
We manage changes via Github 'Pull Requests'. If you're experienced in doing this, then all you need to know is that this repository is available at https://github.com/dxw/playbook2
```

File: _guides/contributing.md [hunk 2]
```diff
@@ -1,143 +1,35 @@
...
+Your suggested changes will not be live yet. Your suggestions should be available to view on the [Github Pull Requests page](https://github.com/dxw/playbook2/pulls). After a few minutes, you should be able to view a preview of the playbook with your changes by clicking on the URL next to "😎 Browse the preview"
 
-When the create or edit steps are followed the result is a pull request, which
-is essentially a request for a change to be added to the Playbook. Merging a
-pull request means accepting a change into the Playbook.
+## Reviewing changes
 
-Anyone who is part of the staff team on Github can approve and merge pull
-requests.
+Everyone's suggested changes that are being reviewed are publicly available. Anyone can publicly comment on a suggested change. Only github accounts that have been added to the dxw Github organisation and staff team can 'merge' a pull request and make it live. If you are not on this list and want to be, please contact dxw ops with your Github username.
```
Comment (rjw1) [inline _guides/contributing.md:31]: Can we say to email support@dxw.com

File: _guides/contributing.md [hunk 3]
```diff
@@ -1,143 +1,35 @@
...
-
-1. [Log in and authenticate with GitHub](https://dxw-playbook-v20.netlify.app/admin)
-2. Open the [Guides collection](https://dxw-playbook-v20.netlify.app/admin/#/collections/guides)
-3. Click "New Guides"
-4. Add your new content using either markdown or rich text
-5. **Save** your change - do **not** Publish or mark as In Review, because we still want to do a collective review of any changes to the Playbook
-
-Saving the change will automatically create a pull request against the Playbook repository.
-
-### Via Github
```
Comment (rjw1) [inline _guides/contributing.md:64]: Can we be consistent in our branding of GitHub.

**Review by rjw1 — CHANGES_REQUESTED** [review]

**Review by rjw1 — APPROVED** [review]

File: _guides/contributing.md [hunk 4]
```diff
@@ -1,143 +1,35 @@
...
-   3. Create a pull request
+   * If it is the homepage, then click 'Index page'
+   * If it is one of the guides, click 'Guides' under 'Collections', and then click the guide you want to edit
+   * If you want to create a new guide, click 'Guides' under 'Collections', then click the 'New Guides' button.
+4. Make your edits in the 'Rich Text' editor.
+5. Click 'Save' at the top of the page
+6. Click 'Status: Draft' and choose 'In review'
 
-## Merging pull requests
+Your suggested changes will not be live yet. Your suggestions should be available to view on the [GitHub Pull Requests page](https://github.com/dxw/playbook2/pulls). After a few minutes, you should be able to view a preview of the playbook with your changes by clicking on the URL next to "😎 Browse the preview"
```
Comment (CristinaRO) [inline _guides/contributing.md:27]: Style consistency questions: we use both single and double quotes on this page. Do we want to choose one and stick with it? In which contexts would the distinction matter?

File: _guides/contributing.md [hunk 5]
```diff
@@ -1,143 +1,35 @@
...
-* it’s something we want to/are comfortable to share outside of dxw.
-* we want it to be visible to the wider team.
-* it's user research or design glossary.
-
-### It’s in guides if
-
-* it’s a detailed, step by step instruction.
-* we’re talking about something specialist. This could mean we are using jargon
-  and referring to community-known methods and ways of working.
+This playbook is public and anyone, inside or outside of dxw, can suggest changes to it. This means you!
```
Comment (CristinaRO) [inline _guides/contributing.md:4]: Style question: I was under the impression that we prefer to capitalise Playbook as a branding thing, is this wrong?

File: _guides/contributing.md (hunk 4 above)
Comment (james) [reply _guides/contributing.md]: I hadn't thought about it, good catch. I've now made everything single quotes.

File: _guides/contributing.md (hunk 5 above)
Comment (james) [reply _guides/contributing.md]: I didn't know that. I have now updated :-)

**Pull request merged**